homeassistant/components/matter/sensor.py

```diff
@@ -384,7 +384,7 @@ def _update_from_device(self) -> None:
             key="ThirdRealityEnergySensorWattAccumulated",
             device_class=SensorDeviceClass.ENERGY,
             entity_category=EntityCategory.DIAGNOSTIC,
-            native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+            native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
             suggested_display_precision=3,
             state_class=SensorStateClass.TOTAL_INCREASING,
             measurement_to_ha=lambda x: x / 1000,
```

Comment:
I am always confused with these conversions. taking the `measurement_to_ha` in account, does that mean the number we get from the device are in mWh?

(looking to see if that conversion is obsolete)